cudamapper/src/index_batcher.cu

```diff
@@ -0,0 +1,134 @@
+/*
+* Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+*
+* NVIDIA CORPORATION and its licensors retain all intellectual property
+* and proprietary rights in and to this software, related documentation
+* and any modifications thereto.  Any use, reproduction, disclosure or
+* distribution of this software and related documentation without an express
+* license agreement from NVIDIA CORPORATION is strictly prohibited.
+*/
+
+#include "index_batcher.cuh"
+
+#include <algorithm>
+#include <exception>
+
+#include <claragenomics/utils/signed_integer_utils.hpp>
```

Comment:
I think this include is not needed anymore.

Reply:
done